### PR TITLE
SEO-178962-MAUI-On-Page-Changes

### PR DESCRIPTION
**Pull request by Scholar-jaruha**
Hi @Gowri-V-L,
Please review the changes made.
Regards,
Scholar.

### CODE REVIEW

**Review by Gowri-V-L — APPROVED** [review]
changes are fine

File: MAUI/Autocomplete/Selection.md [hunk 1]
```diff
@@ -183,7 +183,7 @@ private async void OnSelectionChanged(object sender, Syncfusion.Maui.Inputs.Sele
 
 The following gif image illustrates the result of the above code:
 
-![.NET MAUI Autocomplete](Images/Selection/SelectionChangedEventNotification.png)
+![.NET MAUI Autocomplete selection changed event notification.](Images/Selection/net-maui-autocomplete-selection-changed.png)
```
Comment (Girija-Rajendran) [inline MAUI/Autocomplete/Selection.md:186]: "selection-changed-notification" is the correct term. So, include the notification word in the image name.

File: MAUI/Carousel-View/Transformation.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 layout : post
 title: Transformation in .NET MAUI Carousel View control | Syncfusion
-description: Learn here all about Transformation support in Syncfusion .NET MAUI Carousel View (SfCarousel) control and more.
+description: Learn here all about transformation support in Syncfusion .NET MAUI Carousel View (SfCarousel) control, its elements and more.
```
Comment (Girija-Rajendran) [inline MAUI/Carousel-View/Transformation.md:4]: revert this 
image

File: MAUI/Cartesian-Charts/StackedArea.md [hunk 3]
```diff
@@ -101,4 +101,4 @@ this.Content = chart;
 
 {% endtabs %}
 
-![Stacking Area Chart in MAUI](Chart-types_images\StackedAreaChart.png)
```
Comment (Girija-Rajendran) [inline MAUI/Cartesian-Charts/StackedArea.md:104]: Change the folder format
image

File: MAUI/Autocomplete/Selection.md (hunk 1 above)
Comment (Scholar-jaruha) [reply MAUI/Autocomplete/Selection.md:186]: changed

File: MAUI/Carousel-View/Transformation.md (hunk 2 above)
Comment (Scholar-jaruha) [reply MAUI/Carousel-View/Transformation.md:4]: changed

File: MAUI/Cartesian-Charts/StackedArea.md (hunk 3 above)
Comment (Scholar-jaruha) [reply MAUI/Cartesian-Charts/StackedArea.md:104]: Changed

**Review by Girija-Rajendran — APPROVED** [review]

**Pull request merged**